### PR TITLE
Update Sidecar version to 18.0.0

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
### Description

Update Sidecar version to 18.0.0

### Checklist

- [✔ ] I have added a label to this PR 🏷️
- [ ✔] I have run my changes through Grammarly
- [ ✔] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ✔] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [✔ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ✔] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ✔] If images have been added, I have run the `compress-images.py` script to compress the images.
- [✔ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ✔] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ✔] Will need to create PR in `moonbeam-docs` repo to remove images
- [✔ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ✔] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ✔] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]
LGTM

**Review by eshaben — COMMENTED** [review]
The first image on the sidecar page shows version 14.1.1. It'd be great if this could be changed to a terminal snippet (like [this](https://github.com/moonbeam-foundation/moonbeam-docs/blob/master/.snippets/code/builders/get-started/networks/moonbeam-dev/terminal/docker-pull.md?plain=1)) and updated to show version 18.0.0 please!

File: .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/sidecar.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <br>📦 LOG:
+        <br>    ✅ LEVEL: "info"
+        <br>    ✅ JSON: false
+        <br>    ✅ FILTER_RPC: false
+        <br>    ✅ STRIP_ANSI: false
+        <br>    ✅ WRITE: false
+        <br>    ✅ WRITE_PATH: "/Users/wuzhongzhu/Documents/Work/Foundry/temp/node_modules/@substrate/api-sidecar/build/src/logs"
+        <br>    ✅ WRITE_MAX_FILE_SIZE: 5242880
+        <br>    ✅ WRITE_MAX_FILES: 5
+        📦 SUBSTRATE:
```
Comment (eshaben) [inline .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/sidecar.md:15]:
```suggestion
        <br>📦 SUBSTRATE:
```

File: .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/sidecar.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <br>    ✅ URL: "wss://wss.api.moonbeam.network"
+        <br>    ✅ TYPES_BUNDLE: undefined
+        <br>    ✅ TYPES_CHAIN: undefined
+        <br>    ✅ TYPES_SPEC: undefined
+        <br>    ✅ TYPES: undefined
+        📦 EXPRESS:
```
Comment (eshaben) [inline .snippets/code/builders/get-started/networks/moonbeam-dev/terminal/sidecar.md:21]:
```suggestion
        <br>📦 EXPRESS:
```

**Review by eshaben — APPROVED** [review]
LGTM! Thanks! ✨

**Comment by eshaben** [comment]
@albertov19 your feedback was addressed, can you please approve? 🙂

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**